### PR TITLE
Move collection doc cache version to dedicated file

### PR DESCRIPTION
**Pull request by cidrblock**
This will allow for <s>the</s> a version file to be updated later based on repo tags rather than being hard coded in the project.

### CODE REVIEW

File: src/ansible_navigator/_version_doc_cache.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+"""Version information for ansible-navigator collection doc cache.
+
+.. note::
+
+   __version_collection_doc_cache__:
```
Comment (webknjaz) [inline src/ansible_navigator/_version_doc_cache.py:5]:
```suggestion
   ``__version_collection_doc_cache__``:
```

File: src/ansible_navigator/_version_doc_cache.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+   Indicates the version of the schema of the collection doc cache
+   this is checked during initialization, if the version of the cache
+   differs from below, the cache will be rebuilt.  This should be
+   incremented when the schema changes and need not correspond to the
+   application version, although keeping the major in sync is probably
+   not a bad idea to minimize the amount of stale docs in the user's cache
+"""
+__version_collection_doc_cache__ = "1.0"
```
Comment (webknjaz) [inline src/ansible_navigator/_version_doc_cache.py:14]: Maybe these docstrings should be separate?
```suggestion
"""Version information for ansible-navigator collection doc cache."""

__version_collection_doc_cache__ = "1.0"
"""
Indicates the version of the schema of the collection doc cache
this is checked during initialization, if the version of the cache
differs from below, the cache will be rebuilt.  This should be
incremented when the schema changes and need not correspond to the
application version, although keeping the major in sync is probably
not a bad idea to minimize the amount of stale docs in the user's cache
"""
```

File: src/ansible_navigator/_version_doc_cache.py (hunk 1 above)
Comment (cidrblock) [reply src/ansible_navigator/_version_doc_cache.py:5]: This was previously approved in #651

File: src/ansible_navigator/_version_doc_cache.py (hunk 2 above)
Comment (cidrblock) [reply src/ansible_navigator/_version_doc_cache.py]: I think it's fine as is.

**Comment by cidrblock** [comment]
> I don't think `_version.py` should be reused for this. I'd rather have an additional dedicated `_scm_version.py` file that is gitignored as explained in #553 [#989 (comment)](https://github.com/ansible/ansible-navigator/pull/989#discussion_r811236727).

Description revised to read `a` version file rather than `the` version file.

**Review by webknjaz — APPROVED** [review]

**Pull request merged**